aries-site/src/pages/components/anchor.mdx

```diff
@@ -155,3 +126,38 @@ Used to indicate that an anchor cannot be interacted with.
 >
   <AnchorDisabledExample />
 </Example>
+
+## Accessibility
+### Guidance
+The following guidance should be considered for each instance of the component and is the responsibility of the implementer. WCAG criteria provided directly by the Grommet component are summarized under WCAG Compliance.
+
+**Note**: WCAG rules marked as conditional indicate that the implementer must take additional actions to ensure full compliance.
+
+**Use 'target' and 'rel' props when linking externally:**
+
+If clicking an Anchor leads to an external URL, apply these props to the anchor:
+
+- target="\_blank"
+- rel="noopener"
+
+The distinction is important to screen reader users to know what's going to happen next. Will I navigate somewhere or will something happen on the page?
+
+The "\_blank" target causes the link to open in a new tab, so the user doesn't lose their place on the application. Adding the "noopener" property prevents the newly opened page from gaining access to the original page which helps maintain the security of your application.
+
+**Use clear Anchor labels:**
+
+It is important to make sure the label of the anchor will allow screen reader users to know where the anchor will lead.
+
+Unhelpful Anchor label:
+
+- Read more <GrommetAnchor label="here" />
+
+The word "here" is vague. A screen reader user may not know the context of "here".
+
+Helpful Anchor label:
+
+- Read more <GrommetAnchor label="Anchor documentation" />
+
+This clearly states the type of content to which the Anchor relates.
+
+### WCAG compliance
```

Comment:
What happens if I am a content editor and add content below this in the markdown? This assumes that WCAG compliance will always be the last content in a page.

Having the accessibility section incorporated into `Layout.js` seems too brittle. I'd expect that I'd be importing and placing the WCAG section component here in the markdown.

Reply:
Initially I put it to be rendered in the main layout thinking it would save us from having to edit the mdx files for all of the components but I do see your point here as "insider knowledge" and if we did add anything under it would not work as if it was in the mdx.